biogrid/apps/api/src/app/services/Biogrid.ts

```diff
@@ -40,21 +41,24 @@ function createRandomBuildingPosition(
 export async function simulateNewBiogrid(
   body: NewBiogridOpts
 ): Promise<BiogridSimulationResults> {
-  const buildings = new Array(body.numBuildings).map((v) => {
+  const buildings: Building[] = [];
```

Comment:
create a test for this as well

Reply:
Created! I caught a bug with it as well!